provider/pdns_test.go

```diff
@@ -912,6 +979,25 @@ func (suite *NewPDNSProviderTestSuite) TestPDNSmutateRecords() {
 
 }
 
+func (suite *NewPDNSProviderTestSuite) TestPDNSClientPartitionZones() {
+	// Function definition: PartitionZones(zones []pgo.Zone) (filteredZones []pgo.Zone, residualZones []pgo.Zone)
```

Comment:
Can you drop this line?

Reply:
Sure, was following the convention of the previous test.